src/components/shared/OptionGroup.tsx

```diff
@@ -9,7 +9,7 @@ import {
   searchBoxScrolledStyle,
   searchBoxWrapper,
   optionsWrapper,
-  fixedDiv
+  fixedHeightDiv
```

Comment:
fixedHeightDiv, relativePositionedDiv, divHavingZIndex ?

why can this be searchBoxWrapper ?

Reply:
Removed altogether.